src/index.html

```diff
@@ -5,9 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="prod-card" data-qa="card">
+
+      <img src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      class="prod-card__img"
+      >
```

Comment:
Pay attention on code style
```suggestion
      <img 
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        class="prod-card__img"
      >
```